### PR TITLE
Update router setup and fix links

### PR DESCRIPTION
**Pull request by soltysh**
I:
- fixed SCC setup on the overview page
- updated links 
- removed `--credentials` flag which is superseded by `--service-account` 

@knobunc or somebody from your team ptal for the networking changes
@ahardin-rh ptal for doc itself

### CODE REVIEW

File: install_config/router/default_haproxy_router.adoc [hunk 1]
```diff
@@ -82,23 +78,20 @@ ifdef::openshift-enterprise[]
 The default router service account, named *router*, is automatically created during quick and advanced installations. To verify that this account already exists:
 endif::[]
 ifdef::openshift-origin[]
-First, ensure you have xref:creating-the-router-service-account[created the
+First, ensure you have xref:index.adoc#creating-the-router-service-account[created the
```
Comment (ahardin-rh) [inline install_config/router/default_haproxy_router.adoc]: xref:../install_config/router/index.adoc#creating-the-router-service-account[created the router service account]

File: install_config/router/index.adoc [hunk 2]
```diff
@@ -61,42 +61,44 @@ xref:../../architecture/additional_concepts/authorization.adoc#security-context-
...
 
 
 // NB[1]: The following blurb+codeblock is to both:
-// - Creating the Router Service Account (origin)
+// - Configuring the Router Service Account (origin)
 // - Router Service Account (enterprise)
 // and both of these have anchor ‘creating-the-router-service-account’.
 
-Use of labels (e.g., to define xref:using-router-shards[router shards])
+Use of labels (e.g., to define xref:default_haproxy_router.adoc#using-router-shards[router shards])
```
Comment (ahardin-rh) [inline install_config/router/index.adoc]: xref:../install_config/router/default_haproxy_router.adoc#using-router-shards[router shards])

File: install_config/router/index.adoc [hunk 3]
```diff
@@ -61,42 +61,44 @@ xref:../../architecture/additional_concepts/authorization.adoc#security-context-
...
 requires `cluster-reader` permission.
 
 ----
 $ oadm policy add-cluster-role-to-user \
     cluster-reader \
     system:serviceaccount:default:router
 ----
+
+ifdef::openshift-origin[]
+With a service account in place, you can proceed to installing xref:default_haproxy_router.adoc[a default HAProxy Router],
```
Comment (ahardin-rh) [inline install_config/router/index.adoc]: xref:../install_config/router/default_haproxy_router.adoc#install-config-router-default-haproxy[a default HAProxy Router]

File: install_config/router/index.adoc [hunk 4]
```diff
@@ -61,42 +61,44 @@ xref:../../architecture/additional_concepts/authorization.adoc#security-context-
...
+xref:customized_haproxy_router.adoc[a customized HAProxy Router] or xref:f5_router.adoc[F5 Router].
```
Comment (ahardin-rh) [inline install_config/router/index.adoc]: xref:../install_config/router/customized_haproxy_router.adoc#install-config-router-customized-haproxy[a customized HAProxy Router]

xref:../install_config/router/f5_router.adoc#install-config-router-f5[F5 Router]

**Comment by ahardin-rh** [comment]
@soltysh just a few comments from me regarding links. Is this for 3.3? Thanks!

**Review by knobunc — COMMENTED** [review]
LGTM, @JacobTanenbaum PTAL

File: install_config/router/default_haproxy_router.adoc (hunk 1 above)
Comment (soltysh) [inline install_config/router/default_haproxy_router.adoc]: Interesting, I thought if this is the same directory this should suffice. Fixing now...

**Comment by soltysh** [comment]
@ahardin-rh updated links, waiting for final sing-off from @JacobTanenbaum

**Comment by ahardin-rh** [comment]
@soltysh Thanks! Please check my labels. I am assuming this is for 3.3.

**Comment by soltysh** [comment]
:+1: yup it does make sense to be part of 3.3 Thanks!

**Comment by JacobTanenbaum** [comment]
LGTM

**Comment by ahardin-rh** [comment]
Thanks!

**Comment by vikram-redhat** [comment]
Discussed this with @bfallonf - this was missed for 3.3 because of the to follow up label. Removed that as it doesn't really need a follow up. Marked it for inclusion in the next release.

**Pull request merged**